composer.json

```diff
@@ -7,16 +7,19 @@
     "php": ">=7.1",
     "spryker/calculation-extension": "^1.0.0",
     "spryker/currency": "^3.1.0",
+    "spryker/customer": "^7.25.0",
     "spryker/gui": "^3.17.0",
     "spryker/kernel": "^3.30.0",
     "spryker/money": "^2.0.0",
+    "spryker/price": "^1.0.0",
     "spryker/propel-orm": "^1.0.0",
-    "spryker/sales": "^6.0.0 || ^7.0.0 || ^8.0.0 || ^10.0.0",
+    "spryker/sales": "^11.0.0",
```

Comment:
we need minimum 10 https://spryker.atlassian.net/wiki/spaces/CORE/pages/1849983285/Marketplace+2021+Requirements

is it possible?

Reply:
anyway if we remove previous Sales versions it will be major for Shipment, so it doesn't make sense to do such backport

Reply:
We are trying to put into Shipment v6  three new methods:
    1. ShipmentFacade::saveShipment()
    2. ShipmentFacade::createShipmentGroupTransferWithListedItems()
    3. ShipmentFacade::findShipmentMethodByName()

Currently, in Shipment v6, we have a dependency to  "spryker/sales": "^6.0.0 || ^7.0.0 || ^8.0.0 || ^10.0.0".

In our new `ShipmentFacade` methods, we use the following methods from `SalesFacade`:
    1. SalesFacade::getOrderByIdSalesOrder
    2. SalesFacade::createOrderAddress
    3. SalesFacade::createSalesExpense
    4. SalesFacade::updateSalesExpense
    5. SalesFacade::expandWithCustomerOrSalesAddress

Here is the list of  `Sales` module versions that we need to have backports for:

V6
Exists:
   - SalesFacade::getOrderByIdSalesOrder
Missing:
   - SalesFacade::createOrderAddress 
   - SalesFacade::createSalesExpense() 
   - SalesFacade::updateSalesExpense() 
   - SalesFacade::expandWithCustomerOrSalesAddress() 
V7
Exists:
   - SalesFacade::getOrderByIdSalesOrder
Missing:
   - SalesFacade::createOrderAddress 
   - SalesFacade::createSalesExpense() 
   - SalesFacade::updateSalesExpense() 
   - SalesFacade::expandWithCustomerOrSalesAddress() 

V8
Exists:
   - SalesFacade::getOrderByIdSalesOrder
Missing:
   - SalesFacade::createOrderAddress
   - SalesFacade::createSalesExpense() 
   - SalesFacade::updateSalesExpense() 
   - SalesFacade::expandWithCustomerOrSalesAddress() 

V9
Exists:
   - SalesFacade::getOrderByIdSalesOrder
   - SalesFacade::createSalesExpense() 
Missing:
   - SalesFacade::createOrderAddress 
   - SalesFacade::updateSalesExpense() 
   - SalesFacade::expandWithCustomerOrSalesAddress() 

V10
Exists:
   - SalesFacade::getOrderByIdSalesOrder
   - SalesFacade::createSales	Expense() 
Missing:
   - SalesFacade::createOrderAddress 
   - SalesFacade::updateSalesExpense() 
   - SalesFacade::expandWithCustomerOrSalesAddress() 

This PR, it's just a one example from many we'll face during to adding support for modules from this list https://spryker.atlassian.net/wiki/spaces/CORE/pages/1849983285/Marketplace+2021+Module+Requirements.
Taking into the account that our backports are not fully tested,IMO instead of having full support of old spryker version, we'll have a huge possibility to break the old versions of core modules. Development effort will be huge as well.


@ihorsamusenko do we have a plan for such cases?